exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -292,7 +294,15 @@ private Type getType(MaterializedField field) {
         }
         return new GroupType(dataMode == DataMode.REPEATED ? Repetition.REPEATED : Repetition.OPTIONAL, field.getName(), types);
       case LIST:
-        throw new UnsupportedOperationException("Unsupported type " + minorType);
+        Preconditions.checkArgument(!field.getChildren().isEmpty(),
```

Comment:
Is there a case when this check is needed? If occurrence of such situation is unlikely, it's better to remove if to avoid unnecessary checks during write.